transformer/app.js

```diff
@@ -4,9 +4,11 @@ import { DynamoDBClient, QueryCommand } from '@aws-sdk/client-dynamodb';
 import { getLogger } from './logger.js';
 
 const logger = getLogger();
+const region = process.env.AWS_REGION;
```

Comment:
This variable is redundant since it is only used once in the next line. Just inline it.

Reply:
It's also used on lines 75 and 79. We might remove 79 on the next PR. But anyway, the var should stay.

Reply:
and uppercased.